tests/user/test_user_debug_logging.py

```diff
@@ -184,6 +188,98 @@ def test_add_user_request_debug_log_passphrases_and_passwords_get_redacted_on_er
         self.assertNotIn(self.test_passphrase, error_log)
         self.assertNotIn(self.test_password, error_log)
 
+    def test_add_user_request_debug_log_password_xml_tags_not_redacted_on_success(
+        self,
+        call_racf_mock: Mock,
+    ):
+        call_racf_mock.return_value = (
+            TestUserConstants.TEST_ADD_USER_PASSWORD_RESULT_SUCCESS_XML
+        )
+        stdout = io.StringIO()
+        with contextlib.redirect_stdout(stdout):
+            self.user_admin.add(
+                "squidwrd",
+                traits=TestUserConstants.TEST_ADD_USER_REQUEST_TRAITS_PASSWORD_SIMPLE,
+            )
+        success_log = self.ansi_escape.sub("", stdout.getvalue())
+        self.assertEqual(
+            success_log, TestUserConstants.TEST_ADD_USER_PASSWORD_SUCCESS_LOG
+        )
+        self.assertNotIn("(" + self.simple_password + ")", success_log)
+        self.assertNotIn('"' + self.simple_password + '"', success_log)
+        self.assertNotIn(">" + self.simple_password + "<", success_log)
+        self.assertIn(self.simple_password, success_log)
```

Comment:
I don't know if the `assertNotIn` and `assertIn` statements are necessary here. I think it is enough to just check that the log file still looks like the sample even if you use a password that matches other things in the data.

Reply:
Fair enough, I agree that the existing assert covers that. The intent was that if the log file was updated in some way that these would still be necessary. Fixed now.